yellowbrick/utils/types.py

```diff
@@ -202,6 +202,27 @@ def is_dataframe(obj):
 isdataframe = is_dataframe
 
 
+def is_series(obj):
```

Comment:
Thanks for adding this! It would also be helpful if you wouldn't mind creating some tests for the type checking; the tests are pretty well defined in `tests/test_utils/test_types.py` -- should be a matter of copy and paste.

Reply:
No problem!